specs/albums.spec.js

```diff
@@ -8,34 +8,33 @@ const data = require("../server/data.json");
 describe("ALBUMS", () => {
   describe("CREATE", () => {
     let addedId;
-    it("should add an album", () => {
-      return chakram
-        .post(api.url("albums"), {
-          userId: 1,
-          id: 101,
-          title: "title",
-        })
-        .then(response => {
-          expect(response.response.statusCode).to.match(/^20/);
-          expect(response.body.data.id).to.be.defined;
-          addedId = response.body.data.id;
-          const album = chakram.get(api.url(`albums/${addedId}`));
-          expect(album).to.have.status(200);
-          expect(album).to.have.json("data.id", addedId);
-          expect(album).to.have.json("data.title", "title");
-          expect(album).to.have.json("data.userId", 1);
-          return chakram.wait();
-        });
+    const albumData = {
+      userId: 1,
+      id: 101,
+      title: "title",
+    };
+    it("should add an album", async () => {
+      const originalLength = data.albums.length;
+      const response = await chakram.post(api.url("albums"), albumData);
+      expect(response.response.statusCode).to.match(/^20/);
+      expect(response.body.data.id).to.be.defined;
+      addedId = response.body.data.id;
+      const album = await chakram.get(api.url(`albums/${addedId}`));
+      expect(album).to.have.status(200);
+      expect(album).to.have.json("data.id", addedId);
+      expect(album).to.have.json("data.title", `${albumData.title}`);
+      expect(album).to.have.json("data.userId", albumData.userId);
+      const responseGet = await chakram.get(api.url("albums"));
+      expect(responseGet).to.have.status(200);
+      expect(responseGet).to.have.json("data", albums => {
+        expect(albums).to.be.instanceOf(Array);
+        expect(albums.length).not.to.equal(originalLength);
```

Comment:
It would be a stricter expectation. You can check that a number is greater than the given number.

Reply:
made it stricter